examples/prometheus/prometheus.yaml

```diff
@@ -460,6 +460,14 @@ objects:
           expr: sum(rate(container_cpu_usage_seconds_total{id="/"}[5m])) BY(kubernetes_io_hostname) / ON(kubernetes_io_hostname) machine_cpu_cores
         - record: apiserver_request_count_rate_by_resources
           expr: sum without (client,instance,contentType) (rate(apiserver_request_count[5m]))
+      - name: aggregate_build_metrics
+        rules:
+        - record: build_error_rate
+          expr: sum(openshift_build_total{job="kubernetes-apiservers",phase="Error"})/(sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Failed|Complete|Error"}))
```

Comment:
this was the best i could come up with given the existing build metrics.  Unfortunately it's not super useful.  The problem is that the metric is a gauge based on the current builds in etcd.  Since we prune(by default) builds to the last 5 successful and last 5 failed/errored, in "steady state" you're always going to have a 50/50 ratio of successful+failed builds.  So we can't look at the failed rate.  

In a healthy cluster there should be (nearly) no "error" builds, so we can at least see if that value spikes, but again the error rate will never go higher than 50% because worst case (assuming you had successful builds at some point) you'll always have 5 successful builds and (at worst) 5 error builds, for each buildconfig.

Fixing this would require we introduce a counter for builds(so we maintain historical totals, not just current totals), which we tried to tackle previously but proved challenging due to all the ways that builds can change state (not just in the build_controller).